scripts/build/clean.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @fileoverview Cleans top-level build directories (for OSS and internal processes),
+ * dist subfolders under packages, and .js[.map] files in package directories.
+ */
+
+const {exec} = require('child_process');
+const fs = require('fs');
+
+const {sync: globSync} = require('glob');
+
+if (require.main === module) {
+  main();
+}
+
+function main() {
+  removeDirectory('build');
+  removeDirectory('.rewrite-tmp');
+  removeFilesOfType('css');
+  removeFilesOfType('js');
+  removeFilesOfType('d.ts');
+  removeFilesOfType('map');
+}
+
+function removeDirectory(directory) {
+  exec(`del-cli ${directory}/ ${directory}`, (err) => {
```

Comment:
I vaguely recall running into a situation where `del-cli` would delete all subdirectories and files, but not the parent directory itself, on macOS. Not sure if that's still an issue tho.

Reply:
Ya I was copying what we had before in the package.json. I remember you (ken) saying that we needed it because of Windows. But again not sure if its an issue either. I can test it out on windows and mac and see if just ${directory} works.

Reply:
Yes, but we had `del-cli dir/** dir` before, not `dir/ dir` which AFAIK is exactly the same thing twice.

Reply:
Going to resolve this thread given we're now using `del` and I haven't seen the old problem come up.